Css/aboutMe.css

```diff
@@ -0,0 +1,95 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+html,
+body {
+    max-width: 100%;
+    overflow-x: hidden;
+    background-color: white;
+}
+
+header {
+    padding-top: 400px;
+    background-image: url(/Img/banner.png);
+}
+
+p {
+    font-size: 15px;
+    line-height: 25px;
+}
+
+h1 {
+    text-align: center;
+    margin: 50px 0px 20px 0px;
+}
+
+a {
+    color: #e50914;
+}
+
+.navigation_bar {
+    background-color: black;
+    padding: 20px 0px 20px 0px;
+    display: flex;
+    justify-content: space-around;
+}
+
+.navigation_bar a {
+    text-decoration: none;
+    margin: 15px 0px 0px 0px;
+    font-size: 15px;
+    color: white;
+    font-weight: bold;
+    text-align: center;
+}
+
+.navigation_bar a:hover {
+    text-decoration: underline;
+}
+
+.Image {
+    margin: 0px 0px 40px 200px;
```

Comment:
try to avoid using absolute margins bigger than 80 px, if you want a margin/padding this big, use a relative unit like %